tap/plugin.py

```diff
@@ -25,11 +25,17 @@ def options(self, parser, env=os.environ):
             '--tap-outdir',
             help='An optional output directory to write TAP files to. If the'
                  ' directory does not exist, it will be created.')
+        parser.add_option(
+            '--tap-format',
+            default='',
+            help='An optional format string for the TAP output'
```

Comment:
This will continue in an unexpected way. The space is omitted so the string will look like `... TAP output{sd} ...`. Also, instead of `sd` and `mn`, could you use the whole name? `{short_description}` would be very clear in a format string whereas `{sd}` is not immediately obvious.